nautobot/utilities/tables.py

```diff
@@ -382,3 +398,59 @@ def render(self, record, bound_column, value):
             template = escape(value)
 
         return mark_safe(template)
+
+
+class RelationshipColumn(tables.Column):
+    """
+    Display relationship association instances in the appropriate format.
+    """
+
+    def __init__(self, relationship, side, *args, **kwargs):
+        self.relationship = relationship
+        self.side = side
+        self.peer_side = RelationshipSideChoices.OPPOSITE[side]
+        kwargs["verbose_name"] = relationship.get_label(side)
+        kwargs["accessor"] = Accessor("associations")
```

Comment:
Wonder if these should use setdefault so they can be overridden by the caller if desired?

```suggestion
        kwargs.setdefault("verbose_name", relationship.get_label(side))
        kwargs.setdefault("accessor", Accessor("associations"))
```